Database/3 reports.sql

```diff
@@ -0,0 +1,38 @@
+-- I will need the following reports:
+use ACutAboveDB
+go
+
+-- 1) upoming appointments- seperate report for each hairdresser, ordered by appointment time with more recent appointment 
+-- at the top of the list. Report should be formatted like this: Client first name client last name apt day of the week, apt datetime - service (price) phone number
+select UpcomingAppointments_Sara = CONCAT(a.ClientFirstName, ' ', a.ClientLastName, ' ', a.DayOfWeek,' ', a.AppointmentDate, ' ', a.ServiceReq, ' - ($',a.Price,') ', a.PhoneNumber )
+from Appointments a
+where a.Hairdresser = 'sara'
```

Comment:
I don't think I was clear about this in the spec, but these reports should contain only upcoming appointments.
Can you add a where clause so that all past apts are excluded?